src/test/java/org/broadinstitute/hellbender/engine/AssemblyRegionUnitTest.java

```diff
@@ -122,6 +124,19 @@ private void assertGoodReferenceGetter(final byte[] actualBytes, final SimpleInt
         return tests.subList(2,3).toArray(new Object[][]{});       //HACK!
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Contig chrNotInReference not found in reference.*")
+    //Testing a case where an NPE was thrown when somehow the reference sequence dictionary didn't contain a contig in the
+    //assembly region.
+    public void testMimatchedReferenceAndRegion(){
```

Comment:
`Mimatched` -> `Mismatched`

Reply:
My keyboard is going to kill me with typos.  It's started to just drop random letters and double others.